uber/tests/models/test_job.py

```diff
@@ -68,24 +87,36 @@ def test_overlap(self, session):
 class TestAvailableStaffers:
     @pytest.fixture(autouse=True)
     def extra_setup(self, session, monkeypatch):
-        monkeypatch.setattr(Job, 'all_staffers', [session.staff_one, session.staff_two, session.staff_three, session.staff_four])
+        # note: Assigned Depts, Trusted Depts, and Jobs for this fixture are defined in uber/tests/conftest.py
         monkeypatch.setattr(Job, 'no_overlap', lambda self, a: True)
 
-        session.staff_one.trusted = session.staff_four.trusted = True
-
-        session.staff_one.assigned_depts = str(c.ARCADE)
-        session.staff_two.assigned_depts = str(c.CONSOLE)
-        session.staff_three.assigned_depts = '{},{}'.format(c.ARCADE, c.CONSOLE)
-        session.staff_four.assigned_depts = '{},{}'.format(c.ARCADE, c.CONSOLE)
```

Comment:
the stuff above is pretty different now.  @EliAndrewC I removed the monekypatch() call here and opted to just make our actual test database contain these staffers and have them be assigned/trusted in various departments.